ui/pages/swaps/create-new-swap/create-new-swap.test.js

```diff
@@ -40,13 +40,15 @@ jest.mock('../../../ducks/swaps/swaps', () => {
 });
 
 describe('CreateNewSwap', () => {
+  const props = createProps({ sensitiveTrackingProperties: {} });
```

Comment:
what issue does this resolve?

Reply:
There's a mandatory prop in `CreateNewSwap` component,  https://github.com/MetaMask/metamask-extension/blob/bb2c36396bb00d383f38744791df938f39b2bbf7/ui/pages/swaps/create-new-swap/create-new-swap.js#L46, adding this for the mock